src/dmd/builtin.d

```diff
@@ -381,6 +388,7 @@ public extern (C++) void builtin_init()
     add_builtin("_D3std4math3tanFNaNbNiNeeZe", &eval_tan);
     add_builtin("_D3std4math4sqrtFNaNbNiNeeZe", &eval_sqrt);
     add_builtin("_D3std4math4fabsFNaNbNiNeeZe", &eval_fabs);
+    add_builtin("_D3std4math3expFNaNbNiNeeZe", &eval_exp);
```

Comment:
Could that builtin have gone to druntime instead ? AFAIK that's what we are trying to do nowadays (well, since a few years), and this list should probably be purged since many of those builtin just forward to druntime AFAIK.

Reply:
I think you mixed something up here. These CTFE builtins are generally used for non-CTFEable functions, e.g., if implemented in asm. None of them use druntime, but the host's C functions.